### ISSUE TITLE
32x support 

### ISSUE THREAD

**Issue by Crafyboy** [webrcade/webrcade#115]
It would be pretty cool

**Comment by raz0red** [comment]
Agreed. Looked into it a bit, should be pretty straight-forward to add.

**Comment by mariow-bcp** [comment]
I just added 32X thumbs to my webRcade fork named webRcade new era which will mean the ability to play 32X games will be added later. Once the Team ChuChu! Direct premieres this will be moved to a new studio, named webRcade new era, where the ability to play 32X games will be added later. [Fork Link](https://github.com/mariow-bcp/webrcade-new-era)

**Comment by raz0red** [comment]
Added to 0.2.4-p2
